tests/processors/tests.py

```diff
@@ -177,15 +187,17 @@ def test_sanitize_http_response_cookies(http_test_data):
     )
 
 
-def test_sanitize_http_headers(http_test_data):
+@pytest.mark.parametrize(
+    "elasticapm_client, expected",
+    [
+        ({"sanitize_field_names": "some-value"}, {**_base_expected_headers, "some-value": processors.MASK}),
+        ({"sanitize_field_names": "some-"}, {**_base_expected_headers, "some-value": processors.MASK}),
+        ({"sanitize_field_names": "other-val"}, {**_base_expected_headers, "some-value": "some-secret-value"}),
```

Comment:
Unfortunately, the syntax `{**mydict, "foo": "bar"}` isn't valid in python2. You'll probably just want to keep the `"some-value"` part of the dictionary in the parametrize call and then do a `.update(_base_expected_headers)` in the test itself.